bot/exts/recruitment/talentpool/_review.py

```diff
@@ -180,6 +188,9 @@ async def post_review(self, nomination: Nomination) -> None:
         )
         message = await thread.send(f"<@&{Roles.mod_team}> <@&{Roles.admins}>")
 
+        now = datetime.now(tz=timezone.utc)
```

Comment:
```suggestion
        now = datetime.utcnow()
```

Reply:
that would be incorrect, as it creates a naive datetime object, with the current time offset to UTC. Whereas, what we want here is the current time with UTC as the timezone info.

See the warning in the docs. https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow